pyscriptjs/tests/integration/support.py

```diff
@@ -151,6 +152,9 @@ def writefile(self, filename, content):
         """
         Very thin helper to write a file in the tmpdir
         """
+        file_dir, _ = os.path.split(filename)
+        if file_dir:
+            os.makedirs(file_dir, exist_ok=True)
         f = self.tmpdir.join(filename)
         f.write(content)
```

Comment:
?? f.close()

Reply:
`tmpdir` in `pytest` is of type `<class 'py._path.local.LocalPath'>` and thus it doesn't have a `close` method.
That's why it was never done before also.